main.tf

```diff
@@ -111,7 +112,7 @@ module "container_definition_fluentbit" {
 
 module "service_task" {
   source  = "justtrackio/ecs-scheduled-task/aws"
-  version = "1.1.0"
+  version = "1.2.0"
```

Comment:
This changeset can be found here: https://github.com/justtrackio/terraform-aws-ecs-scheduled-task/pull/16